examples/java/src/main/java/org/apache/beam/examples/common/ExampleUtils.java

```diff
@@ -281,21 +276,7 @@ private void deletePubsubSubscription(String subscription) throws IOException {
   }
 
   /**
-   * Do some runner setup: check that the DirectRunner is not used in conjunction with
-   * streaming, and if streaming is specified, use the DataflowRunner.
-   */
-  private void setupRunner() {
-    Class<? extends PipelineRunner<?>> runner = options.getRunner();
-    if (options.as(StreamingOptions.class).isStreaming()
-        && runner.equals(BlockingDataflowRunner.class)) {
-      // In order to cancel the pipelines automatically,
-      // {@literal DataflowRunner} is forced to be used.
-      options.setRunner(DataflowRunner.class);
-    }
-  }
-
-  /**
-   * If {@literal DataflowRunner} or {@literal BlockingDataflowRunner} is used,
+   * If {@literal DataflowRunner} is used,
```

Comment:
Why is this restricted to the DatfalowRunner? I thought all PipelineResult objects allowed "waitUntilFinish"?